### DISCUSSION TITLE
Nvidia AR SDK Read.me "instructions"

### DISCUSSION THREAD

**Discussion by AWholeTurkey** [patrikhuber/eos#335] [Q&A] [answered]
Hey, I am trying to follow the Nvidia AR SDK ream.me instructions and I cant seem to download two of you json files. (sfm_3448_edge_topology.json and sfm_model_contours.json). One doesn't have the download option, and the other has the option but only ever directs me to the view page for the file. Then I tried to go through the desktop app but I can't clone the repository because it "fails to authenticate"

**Comment by patrikhuber** [comment, marked as answer]
Apologies for the very late reply, I never got a notification of your post or must have missed it. In any case: You can open the file on GitHub.com (e.g. https://github.com/patrikhuber/eos/blob/master/share/sfm_model_contours.json) and then there's a "Raw" button on the top-right.
![image](https://user-images.githubusercontent.com/4967343/228054330-e020e66f-db5f-4daf-a2f1-e263a13b0431.png)

If you weren't able to clone the repo using the GitHub desktop app, that would be a problem with your local setup, not eos.

Alternatively you could view the file on the web, just select all the content with the mouse, and paste it into a text file locally. That's the worst of all options of course :-).
